pkg/chunk/aws/s3_storage_client.go

```diff
@@ -47,6 +47,12 @@ func NewS3ObjectClient(cfg StorageConfig, schemaCfg chunk.SchemaConfig) (chunk.O
 		return nil, err
 	}
 
+	if cfg.S3ForceTLS {
+		if strings.Contains(cfg.S3.URL.Host, ".") {
```

Comment:
Should this check for presence of http/https in the url already?

Reply:
I tried to keep it consistent with how `ConfigFromURL` sets the endpoint which is almost identical to that if statement, but with https instead